src/client/dfuse/dfuse.h

```diff
@@ -24,6 +24,10 @@
 
 #include "dfuse_common.h"
 
+/* TODO: This should update the container open flags once the update has landed
+ * https://github.com/daos-stack/daos/pull/13702
+ */
+
```

Comment:
So we we're wanting to land #13702 first, then rebase this PR?

Reply:
That's my hope.  I could make the change in this PR but it'll conflict.

Reply:
Ok, updated this PR, it'll just have to conflict.